### PR TITLE
fava: 1.7 -> 1.9

### PR DESCRIPTION
**Pull request by elohmeier**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by worldofpeace** [comment]
@GrahamcOfBorg build fava

File: pkgs/applications/office/fava/default.nix [hunk 1]
```diff
@@ -5,18 +5,18 @@ let
...
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "c4eba4203bddaa7bc9d54971d2afeeebab0bc80ce89be1375a41a07c4e82b62f";
+    sha256 = "115r99l6xfliafgkpcf0mndqrvijix5mflg2i56s7xwqr3ch8z9k";
   };
 
   doCheck = false;
 
   propagatedBuildInputs = with python3.pkgs;
```
Comment (worldofpeace) [inline pkgs/applications/office/fava/default.nix]: I'm not seeing all of these declared https://github.com/beancount/fava/blob/088d6ba091d718fc42b21051fa35894fbfb22d23/setup.py#L33

Comment (elohmeier) [reply pkgs/applications/office/fava/default.nix]: Thanks for your review! I've added the missing dependencies.

**Review by worldofpeace — APPROVED** [review]
Built and executed locally with success.

**Comment by worldofpeace** [comment]
Thanks @elohmeier :sparkles:

**Pull request merged**